destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/DefaultStreamLoader.java

```diff
@@ -97,7 +155,15 @@ public StreamLoadResponse send(List<AirbyteRecordMessage> records) throws Except
             throw new IOException("Could not find an available fe host.");
         }
 
-        String sendUrl = String.format("http://%s:%d%s", host, properties.getHttpPort(), loadUrlPath);
+        int port = properties.getHttpPort();
+        String scheme = "http";
```

Comment:
Is it necessary to make the "http scheme" configurable as we may use different port for TLS connection

Reply:
That is a good idea, I will work on that

Reply:
For now, hardcoding the port is a temporary solution. We could refactor the code at a later point in time.